ibm_catalog.json

```
@@ -0,0 +1,122 @@
+{
+    "products": [
+        {
+            "features": [
+                {
+                    "title": "Supports multi-factor authentication",
+                    "description": "Supports multi-factor authentication (MFA) for all users or federated users. Includes email-based, TOTP, and U2F MFA."
+                },
+                {
+                    "title": "Supports general IAM settings",
+                    "description": "User can configure the following IAM Account settings:\n1. Restrict API Key creation. \n2. Restrict service ID creation\n 3. Customize session activity and inactivity timeouts (in seconds). \n4. Customize Access and Refresh token expiration (in seconds). \n5. Global Shell Settings. \n6. Public Access Group."
```

```suggestion
                    "description": "Configure the following IAM settings:\n1. Restrict whether API keys can be created. \n2. Restrict whether service IDs can be created.\n 3. Customize session activity and inactivity timeouts. \n4. Customize the expiration time of both access and refresh token. \n5. Global Shell Settings. \n6. Public Access Group."
```

Can you add more to Global Shell Settings and Public Access Group? What can you do? Also, these are not proper nouns or products, so use lower case (`global shell settings` and `public access group`).